tests/orquestra/integrations/cirq/conversions/circuit_conversions_test.py

```diff
@@ -337,3 +337,27 @@ def test_with_cirq_only_gates_with_free_symbols_check(self, cirq_circuit):
     def test_with_unsupported_gates_raises_not_implemented_error(self, cirq_circuit):
         with pytest.raises(NotImplementedError):
             import_from_cirq(cirq_circuit)
+
+    def test_only_named_qubits_are_converted_to_labeled_qubit(self):
```

Comment:
Circuit in this test has only named qubits, so I think you switched the names of the tests?

Also, I'd like to see some edge cases like circuit with qubits:
[20, "b"], ["a", 0, "b"] (order shouldn't matter, right?)

Reply:
nope! order shouldn't matter. Will add those test cases later tonight.

Reply:
Done!